package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dnlup/fastify-traps",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
There is no need to change this, it will be changed automatically by the release flow in the CI. I order to do so, there must be a 'BREAKING CHANGE: description' line in the commit body. A commit example:

```
feat: support fastify v4

Add support for Fastify version 4.
BREAKING CHANGE: Fastify version 3 is no longer supported
```

If you could amend the commit message like this it would be awesome 😉

Reply:
Hi @dnlup,
Thank you!
Indeed, Fastify v4 dropped the support of Node 12.
I've amended the commit, is there anything else I missed?

Reply:
Thank you for changing the message. A couple of things are missing:

*  the `version` field in `package.json` should remain `1.2.0` (it is handled automatically by the release process)
* we can remove `12.x` from [this matrix in the tests](https://github.com/dnlup/fastify-traps/blob/574a980501d8b8b3a5a6f616fbeec04ff6332d64/.github/workflows/tests.yml#L19)

Reply:
Thank you!
Yeah, I didn't think it's necessary to revert the version in package.json. I pushed the requested changes